server/views/pages/appointments/create-and-edit/schedule.njk

```diff
@@ -197,15 +197,17 @@
                         {% else %}
                             <div class="govuk-summary-card appointment-schedule">
                                 <div class="govuk-summary-card__title-wrapper">
-                                    <h2 class="govuk-summary-card__title" data-qa="schedule-card-title-prison-number-{{ prisoner.number }}">
+                                    <h3 class="govuk-summary-card__title" data-qa="schedule-card-title-prison-number-{{ prisoner.number }}">
```

Comment:
Changed to a h3 since this isn't a new section and this content sits below the h2 above it in the content hierarchy